Python/flowgraph.c

```diff
@@ -2246,8 +2256,8 @@ convert_pseudo_ops(basicblock *entryblock)
 }
 
 static inline bool
-is_exit_without_lineno(basicblock *b) {
-    if (!basicblock_exits_scope(b)) {
+is_exit_or_eval_check_without_lineno(basicblock *b) {
+    if (! (basicblock_exits_scope(b) || basicblock_has_eval_break(b))) {
```

Comment:
Negations around ands/ors are a bit hard to follow. Maybe just add another if block for `!basicblock_has_eval_break(b)` after this one?